app/Http/Traits/HasConversionUnit.php

```
@@ -4,22 +4,52 @@
 
 use App\Http\Services\UomConversionService;
 use App\Models\UOM;
+use InvalidArgumentException;
 
 trait HasConversionUnit
 {
     private $uomIdColumn = 'uom_id';
     private $quantityColumn = 'quantity';
     private $customConversionColumn = 'uom_conversion';
```

PHPMD (2.15.0)

Avoid excessively long variable names like $customConversionColumn. Keep variable name length under 20. (Naming Rules)

(LongVariable)